src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NugetApiPackageManager.cs

```diff
@@ -455,12 +461,13 @@ private IEnumerable<PackageSource> LoadNuGetSources(IEnumerable<string> addition
 
         private class NugetPackageMetadata
         {
-            public NugetPackageMetadata(IPackageSearchMetadata metadata, string owners, bool trusted)
+            public NugetPackageMetadata(IPackageSearchMetadata metadata, string owners, bool trusted, IEnumerable<PackageVulnerabilityMetadata> vulnerabilities)
```

Comment:
```suggestion
            public NugetPackageMetadata(IPackageSearchMetadata metadata, string owners, bool trusted, IReadOnlyList <PackageVulnerabilityMetadata> vulnerabilities)
```

Reply:
The NuGet api returns the `IEnumerable` type, I feel like it is ok to keep using that type in this case.